packages/server/src/db/index.js

```diff
@@ -74,6 +74,10 @@ async function createUser(user) {
     };
 }
 
+async function getUsersEmailAndName(ids) {
+    return knex.select('id', 'name', 'email').from('users').whereIn('id', ids);
```

Comment:
I believe we'll need to add an `await` here since this query will be run async. Can you confirm?

Reply:
Wouldn't prefacing calling the endpoint with await produce the same output? I just used examples set by other endpoints but can change it